kotlin/internal/jvm/compile.bzl

```diff
@@ -175,6 +175,7 @@ def _fold_jars_action(ctx, rule_kind, toolchains, output_jar, input_jars, action
         "--normalize",
         "--compression",
         "--exclude_build_data",
+        "--add_missing_directories",
```

Comment:
Man, I'm confused about this PR... is the fundamental fix adding this flag, or adding the closeable that forces execution? Or is the closeable just do you can use it more conveniently in tests in a `try-with-resources`/`Closeable.use{}`? 

Anyway, I'd laugh/cry if this flag were the fix...  Sigh.

Reply:
this flag is the fix, but I was adding tests to all of 'em.

The closeable just makes the code read cleaner.